web/common.inc.sh

```diff
@@ -105,25 +105,13 @@ function test-headless() {
   fi
```

Comment:
There's still `TEST_CD_REQD=false` in line 102. Unfortunately since it's outside of the context of this hunk I can't comment on that line and suggest a change.